docs/_articles/en/vscode-desktop/java-setup.md

```diff
@@ -12,31 +12,28 @@ You can download the JDK from one of these locations.
 
 [Adoptium](https://adoptium.net) provides prebuilt OpenJDK binaries for free without authentication or complications.
 
-1. Navigate to [Adoptium](https://adoptium.net/?variant=openjdk17).
-2. Select **Temurin 17 (LTS)**.
-3. Click the **Latest Release** button to download.
-4. After the file has downloaded, open it and complete the installation steps.
+1. Navigate to the [Adoptium] (https://adoptium.net/?variant=openjdk21) JDK 21 download page.
+2. Select the installer applicable to your system.
+3. Download and open the downloaded file and complete the installation steps.
 
 ### Zulu
 
 Zulu Java builds are TCK-tested and free to download and use without restrictions.
 
-1. Navigate to the [Zulu download page](https://www.azul.com/downloads/zulu/).
-2. Click the **Choose Your Download** button.
-3. Select Java Version **Java 17 (LTS)**.
-4. Select your OS.
-5. Select your Architecture.
-6. Select your Java Package (JDK).
-7. Download the installer.
-8. Once the file is downloaded open it and complete the installation steps.
+1. Navigate to the [Zulu download page](https://www.azul.com/downloads/?package=jdk#zulu).
```

Comment:
nit: can we point this also to jdk 21 download page directly like the Oracle and Adoptium?
https://www.azul.com/downloads/?version=java-21-lts#zulu